src/mtype.d

```diff
@@ -4747,7 +4747,8 @@ extern (C++) final class TypeSArray : TypeArray
             if (d1 != d2)
             {
             Loverflow:
-                error(loc, "Static array size too big", toChars(), cast(ulong)tbn.size(loc), cast(ulong)d1);
+                auto static_arr_size = tbn.size(loc) * d2;
```

Comment:
Ah, I'm sorry. I didn't realize `overflow` is a by-ref parameter, doh. I do think it's better to keep the exact code `auto static_arr_size = mulu(tbn.size(loc), d2, overflow);`, such that the overflowing is certain to be what it should be.